builtin/logical/consul/path_token.go

```diff
@@ -90,26 +108,43 @@ func (b *backend) pathTokenRead(ctx context.Context, req *logical.Request, d *fr
 	}
 
 	// Create an ACLToken for Consul 1.4 and above
-	policyLink := []*api.ACLTokenPolicyLink{}
-	for _, policyName := range result.Policies {
-		policyLink = append(policyLink, &api.ACLTokenPolicyLink{
-			Name: policyName,
-		})
+	// If policies were supplied here, then overwrite the policies
+	// that were given when the role was written
+	var policyLink []*api.ACLTokenPolicyLink
```

Comment:
A bit of an uber nit, and a pre-existing name, but IME this sort of nit helps readability a bit :) it's easier to remember/infer further down that the type is a slice if it's name is plural

```suggestion
	var policyLinks []*api.ACLTokenPolicyLink
```

Reply:
Yeah I reused the name already present without thinking too much about it, but I'm glad you pointed it out because I agree with you.